pyproject.toml

```diff
@@ -2,220 +2,220 @@
 name = "connect-python"
 version = "0.5.0"
 description = "Server and client runtime library for Connect RPC"
-maintainers = [
-    { name = "Anuraag Agrawal", email = "anuraaga@gmail.com" },
-    { name = "Spencer Nelson", email = "spencer@firetiger.com" },
-    { name = "Stefan VanBuren", email = "svanburen@buf.build" },
-    { name = "Yasushi Itoh", email = "i2y.may.roku@gmail.com" },
-]
-requires-python = ">= 3.10"
-dependencies = ["httpx", "protobuf>=5.28"]
 readme = "README.md"
+requires-python = ">= 3.10"
 license-files = ["LICENSE"]
-keywords = ["rpc", "grpc", "connect", "protobuf", "http"]
+maintainers = [
+  { name = "Anuraag Agrawal", email = "anuraaga@gmail.com" },
+  { name = "Spencer Nelson", email = "spencer@firetiger.com" },
+  { name = "Stefan VanBuren", email = "svanburen@buf.build" },
+  { name = "Yasushi Itoh", email = "i2y.may.roku@gmail.com" },
+]
+keywords = ["connect", "grpc", "http", "protobuf", "rpc"]
 classifiers = [
-    "Development Status :: 3 - Alpha",
-    "Intended Audience :: Developers",
-    "License :: OSI Approved :: Apache Software License",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
-    "Topic :: Internet :: WWW/HTTP",
-    "Topic :: Software Development :: Libraries :: Python Modules",
-    "Typing :: Typed",
+  "Development Status :: 3 - Alpha",
```

Comment:
Just a double check, FWICT the indent change is from bumping ruff to 0.13. Not a problem, I get the same formatting when checking out the PR and just wanted to make sure it's not an environment specific change.

Reply:
ugh, no, not a `ruff` change. sorry, have [tombi](https://github.com/tombi-toml/tombi) enabled locally and didn't even notice it was changing things 🤦 . Let me revert that to not confuse things.